orchestration/scripts/diff-hca-table.sh

```diff
@@ -0,0 +1,49 @@
+set -euo pipefail
+
+# Copy-pasted from StackOverflow.
+function join_by { local d=$1; shift; echo -n "$1"; shift; printf "%s" "${@/#/$d}"; }
+
+# Point to BQ metadata we expect to be present on disk.
+declare -r TABLE_DIR=/bq-metadata/${TABLE}
```

Comment:
In general, all `${VAR}` should be `"${VAR}"` to avoid issues with spaces (although the queries below are OK since they're already in double quotes). In this case I'm guessing TABLE etc don't have spaces but I usually try to do it anyway as a good bash practice.